rpm/0023-sailfishos-gecko-Force-to-build-mozglue-and-xpcomglu.patch

```diff
@@ -0,0 +1,57 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
```

Comment:
This patch looks a bit suspicions actually as sha1 408fd387 was enough for esr60.